### PR TITLE
feat: New spec "rhui-set-release" and parser

### PR DESCRIPTION
**Pull request by huali027**
### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### CODE REVIEW

**Comment by huali027** [comment]
@xiangce @JoySnow Please help to review, thanks.

**Review by JoySnow — CHANGES_REQUESTED** [review]
Hi @huali027 , I am good with the changes.  Just missing a tiny update of docstings:

**Comment by JoySnow** [comment]
About this parser `SubscriptionManagerReleaseShow`, its spec `Specs.subscription_manager_release_show` was no more collected. ( https://gitlab.cee.redhat.com/insights-release-eng/insights-core-assets/-/commit/5a1e057bac0d402b290eb17a6100ca61f78b1370 )  

Thinking about deleting the including of parser `SubscriptionManagerReleaseShow` in it's combiner. 

Well, found that this `Specs.subscription_manager_release_show` is still widely used in Advisor rules' test files. 

Okay, we should keep including this `SubscriptionManagerReleaseShow` here at least for now. 

---- 

BTW, should we consider to mark `deprecation` of parser `SubscriptionManagerReleaseShow` ?  @xiangce

**Comment by xiangce** [comment]
> About this parser `SubscriptionManagerReleaseShow`, its spec `Specs.subscription_manager_release_show` was no more collected. ( https://gitlab.cee.redhat.com/insights-release-eng/insights-core-assets/-/commit/5a1e057bac0d402b290eb17a6100ca61f78b1370 )
> 
> Thinking about deleting the including of parser `SubscriptionManagerReleaseShow` in it's combiner.
> 
> Well, found that this `Specs.subscription_manager_release_show` is still widely used in Advisor rules' test files.
> 
> Okay, we should keep including this `SubscriptionManagerReleaseShow` here at least for now.
> 
> BTW, should we consider to mark `deprecation` of parser `SubscriptionManagerReleaseShow` ? @xiangce

@JoySnow - Yeap, let's deprecate it and update the rules accordingly

**Comment by JoySnow** [comment]
Opened a new thread - [RHINENG-7882](https://issues.redhat.com/browse/RHINENG-7882) to track the parser deprecation. @xiangce 

> > About this parser `SubscriptionManagerReleaseShow`, its spec `Specs.subscription_manager_release_show` was no more collected. ( https://gitlab.cee.redhat.com/insights-release-eng/insights-core-assets/-/commit/5a1e057bac0d402b290eb17a6100ca61f78b1370 )
> > Thinking about deleting the including of parser `SubscriptionManagerReleaseShow` in it's combiner.
> > Well, found that this `Specs.subscription_manager_release_show` is still widely used in Advisor rules' test files.
> > Okay, we should keep including this `SubscriptionManagerReleaseShow` here at least for now.
> > BTW, should we consider to mark `deprecation` of parser `SubscriptionManagerReleaseShow` ? @xiangce
> 
> @JoySnow - Yeap, let's deprecate it and update the rules accordingly

**Review by JoySnow — APPROVED** [review]

**Comment by xiangce** [comment]
@huali027 - please fix the conflicts

File: insights/specs/__init__.py [hunk 1]
```diff
@@ -621,6 +621,7 @@ class Specs(SpecSet):
     rhsm_katello_default_ca_cert = RegistryPoint()
     rhsm_log = RegistryPoint(filterable=True)
     rhsm_releasever = RegistryPoint(no_obfuscate=['hostname', 'ip'])
+    rhui_set_release = RegistryPoint()
```
Comment (xiangce) [inline insights/specs/__init__.py:624]: Per the spec application MR in the assets repo, this spec should be marked as `no_obfuscate=['hostname', 'ip']`

**Review by xiangce — APPROVED** [review]

**Pull request merged**